src/java.base/share/classes/java/text/CompactNumberFormat.java

```diff
@@ -564,6 +564,35 @@ public final StringBuffer format(Object number,
         }
     }
 
+    @Override
+    StringBuilder format(Object number,
+                                     StringBuilder toAppendTo,
```

Comment:
nit: indentation seems odd here/not consistent with the other corresponding method.